Geometry/VeryForwardData/data/CTPPS_2021/RP_Dist_Beam_Cent.xml

```diff
@@ -0,0 +1,49 @@
+	time: 1464650000
+	offset file: /afs/cern.ch/exp/totem/scratch/data/RP/tools/measurement_offsets/zero_offsets
+	sensor type: RESOLVER
+-->
+	<ConstantsSection label="RP_Dist_Beam_Cent.xml">
+		<!-- sector 45, 210m station -->
+		<Constant name="RP_147_Left_Det_Dist_0" value=" +9.129*mm"/> <!-- RESOLVER:  +9.129, offset:  +0.000 -->
+		<Constant name="RP_147_Left_Det_Dist_1" value=" +8.885*mm"/> <!-- RESOLVER:  +8.885, offset:  +0.000 -->
+		<Constant name="RP_147_Left_Det_Dist_2" value=" +6.480*mm"/> <!-- RESOLVER:  +6.480, offset:  +0.000 -->
+		<Constant name="RP_147_Left_Det_Dist_3" value=" +6.181*mm"/> <!-- RESOLVER:  +6.181, offset:  +0.000 -->
+		<Constant name="RP_147_Left_Det_Dist_4" value=" +7.000*mm"/>
+		<Constant name="RP_147_Left_Det_Dist_5" value=" +7.000*mm"/>
+
+		<!-- sector 45, 220m station -->
+		<Constant name="RP_220_Left_Det_Dist_0" value="+43.391*mm"/> <!-- RESOLVER: +43.391, offset:  +0.000 -->
+		<Constant name="RP_220_Left_Det_Dist_1" value="+42.712*mm"/> <!-- RESOLVER: +42.712, offset:  +0.000 -->
+		<!--<Constant name="RP_220_Left_Det_Dist_2" value="+41.915*mm"/>  RESOLVER: +41.915, offset:  +0.000 -->
+		<Constant name="RP_220_Left_Det_Dist_2" value="+3.0*mm"/> <!-- RESOLVER: +41.915, offset:  +0.000 -->
+		<Constant name="RP_220_Left_Det_Dist_3" value="+43.162*mm"/> <!-- RESOLVER: +43.162, offset:  +0.000 -->
+		<Constant name="RP_220_Left_Det_Dist_4" value=" +7.000*mm"/>
+		<Constant name="RP_220_Left_Det_Dist_5" value=" +7.000*mm"/>
+
+		<!-- sector 56, 210m station -->
+		<Constant name="RP_147_Right_Det_Dist_0" value=" +9.052*mm"/> <!-- RESOLVER:  +9.052, offset:  +0.000 -->
+		<Constant name="RP_147_Right_Det_Dist_1" value=" +8.017*mm"/> <!-- RESOLVER:  +8.017, offset:  +0.000 -->
+		<Constant name="RP_147_Right_Det_Dist_2" value=" +6.605*mm"/> <!-- RESOLVER:  +6.605, offset:  +0.000 -->
+		<Constant name="RP_147_Right_Det_Dist_3" value=" +5.319*mm"/> <!-- RESOLVER:  +5.319, offset:  +0.000 -->
+		<Constant name="RP_147_Right_Det_Dist_4" value=" +7.000*mm"/>
+		<Constant name="RP_147_Right_Det_Dist_5" value=" +7.000*mm"/>
+
+		<!-- sector 56, 220m station -->
+		<Constant name="RP_220_Right_Det_Dist_0" value="+42.318*mm"/> <!-- RESOLVER: +42.318, offset:  +0.000 -->
+		<Constant name="RP_220_Right_Det_Dist_1" value="+42.266*mm"/> <!-- RESOLVER: +42.266, offset:  +0.000 -->
+		<!--<Constant name="RP_220_Right_Det_Dist_2" value="+44.364*mm"/>  RESOLVER: +44.364, offset:  +0.000 -->
+		<Constant name="RP_220_Right_Det_Dist_2" value="+3.0*mm"/> <!-- RESOLVER: +44.364, offset:  +0.000 -->
+		<Constant name="RP_220_Right_Det_Dist_3" value="+43.396*mm"/> <!-- RESOLVER: +43.396, offset:  +0.000 -->
+		<Constant name="RP_220_Right_Det_Dist_4" value=" +7.000*mm"/>
+		<Constant name="RP_220_Right_Det_Dist_5" value=" +7.000*mm"/>
+
+    <!-- sector 45, diamond station -->
+    <Constant name="CTPPS_45_Det_Dist" value="3.0*mm"/>
+
+    <!-- sector 56, diamond station -->
```

Comment:
If you edit the file, you may
 * fix the indentation
 * remove unused constants
 * "merge" the timing constants to the blocks above

Reply:
I think it should be done now.